src/decorators/SpaceAround/SpaceAround.native.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import styled from 'styled-components';
```

Comment:
I would suggest let's move it to `storybook/native/index.js` itself since it's not going to be used by any other component. No point of abstracting it

Reply:
Yeah makes more sense